### ISSUE TITLE
Order meta

### ISSUE THREAD

**Issue by erikdemarco** [rayrutjes/wc-order-search-admin#98]
Why it not searching order meta data?

**Comment by rayrutjes** [comment]
This plugin attempts to be as plug & play as possible.
In that effort, we automatically index common data across common use cases.

I'm open to consider extending the lists of default values we push to Algolia.

However we won't send everything by default, as this might make us reach the limits of the record size of 10kb.

Let me know what kind of fields you are interested in seing indexed.

**Comment by erikdemarco** [comment]
Maybe add "custom field" in the setting so user can index what they want themself. Because 1000 user will have 1000 different needs as well.

**Comment by rayrutjes** [comment]
So far, we've managed to provide a plug & play experience for all use cases.
What exactly are you trying to achieve? Maybe we can consider adding it for you.

**Comment by rayrutjes** [comment]
Are you using Advanced Custom Field? (ACF)

**Comment by erikdemarco** [comment]
no, im using custom order meta via php. see this: https://docs.woocommerce.com/wp-content/uploads/2012/04/checkout_field_custom_field_admin.png

**Comment by rayrutjes** [comment]
Hey there.
I'm currently looking to possibly re-vive this plugin if interest is there. Might look into this compat.

If you're still using this plugin or interested in solving the pain it was solving, please checkout
https://github.com/rayrutjes/wc-order-search-admin/issues/137
